app/api/resources/user.py

```diff
@@ -320,7 +320,7 @@ def get(cls, token):
 
 @users_ns.route("user/resend_email")
 @users_ns.response(HTTPStatus.OK, "%s" % messages.EMAIL_VERIFICATION_MESSAGE)
-@users_ns.response(HTTPStatus.BAD_REQUEST, "Invalid input.")
+@users_ns.response(HTTPStatus.BAD_REQUEST, "%s" % messages.EMAIL_INPUT_BY_USER_IS_INVALID)
```

Comment:
```suggestion
@users_ns.response(HTTPStatus.BAD_REQUEST, f"{messages.EMAIL_INPUT_BY_USER_IS_INVALID})
```
I would suggest this, because in this project we are trying to be consistent with f-strings :)